pkgs/common-updater/scripts/list-directory-versions

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import argparse
+import requests
+import os
+import subprocess
+import re
+from bs4 import BeautifulSoup
+
+parser = argparse.ArgumentParser(description="lists all available versions listed for a package in a site.")
+
+os.environ.get('UPDATE_NIX_PNAME', '')
+
+parser.add_argument("--pname", default=os.environ.get('UPDATE_NIX_PNAME', ''), help="Name of the package.")
```

Comment:
I would go with `os.environ.get("UPDATE_NIX_PNAME", None)` and make it fail more loudly. Also you used single quotes inconsistently with the rest here, maybe run `nix-shell -p black --run 'black pkgs/common-updater/scripts/list-directory-versions'` to enforce the formatting.

Also maybe add `required="UPDATE_NIX_PNAME" not in os.environ` kwarg to fail with a nicer message.

Reply:
Done